llama/alma.py

```diff
@@ -1,3 +1,5 @@
+import datetime
```

Comment:
This should be `from datetime import datetime` to make the type hint is accurate (right now it's hinting that the type is the entire `datetime` module :)

Reply:
`datetime`'s duplicative naming is rather obnoxious to me : ) Fixed!